dlrs/main/classes/RollupSummariesSelectorTest.cls-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>52.0</apiVersion>
```

Comment:
Andy suggested keeping this at version 51.0 for now - same as `RollupSummariesSelector.cls`